src/Tests/Microsoft.NET.Sdk.WorkloadManifestReader.Tests/SdkDirectoryWorkloadManifestProviderTests.cs

```diff
@@ -86,21 +88,80 @@ var sdkDirectoryWorkloadManifestProvider
         }
 
         [Fact]
-        public void GivenNoManifestJsonFileInDirectoryItShouldThrow()
+        public void GivenNoManifestJsonFileInDirectoryItShouldIgnoreIt()
         {
             Initialize();
 
-            Directory.CreateDirectory(Path.Combine(_manifestDirectory, "Android"));
+            Directory.CreateDirectory(Path.Combine(_manifestVersionBandDirectory, "Android"));
 
             var sdkDirectoryWorkloadManifestProvider
                 = new SdkDirectoryWorkloadManifestProvider(sdkRootPath: _fakeDotnetRootDirectory, sdkVersion: "5.0.100", userProfileDir: null);
 
-            Action a = () => sdkDirectoryWorkloadManifestProvider.GetManifests().Select(m => {
-                using (m.OpenManifestStream()) { }
-                return true;
-            }).ToList();
+            sdkDirectoryWorkloadManifestProvider.GetManifests()
+                .Should()
+                .BeEmpty();
+        }
+
+        [Fact]
+        public void ItReturnsLatestManifestVersion()
+        {
+            Initialize();
 
-            a.Should().Throw<FileNotFoundException>();
+            CreateMockManifest(_manifestRoot, "5.0.100", "ios", "11.0.1", true);
```

Comment:
Just to safe, should we have a test that use a mixture of stable/non-stable versions?

```cs
[InlineData("11.0.1", "11.0.2", ...)]
[InlineData("11.0.1", "11.0.2-preview", ...)]
```

and one with SxS feature bands, e.g. `5.0.100-preview3\ios\11.0.2` and `5.0.100-preview4\ios\11.02`?